src/main/java/com/server/nodak/domain/s3/validator/ImagePath.java

```diff
@@ -0,0 +1,5 @@
+package com.server.nodak.domain.s3.validator;
+
+public enum ImagePath {
+    userProfile, post
```

Comment:
```suggestion
    USER_PROFILE, POST
```
굳이 상관은 없지만, enum의 내부 값은 일반적으로 모두 대문자, 뛰어쓰기가 필요한 문자는 _를 붙이는 편이라고 알고 있습니다.
나중에 리팩토링이나 편하실때 변경해도 좋을 것 같아용 :)